examples/demo-auth/atmos.yaml

```
@@ -0,0 +1,83 @@
+    - "**/*"
+  excluded_paths:
+    - "**/_defaults.yaml"
+  name_pattern: "{stage}"
+
+logs:
+  file: "/dev/stderr"
+  level: Debug
+
+# Custom CLI commands
+commands:
+- name: "test"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)